package.json

```
@@ -63,7 +63,8 @@
     "projectType": "react",
     "tests": {
       "_comment": "Add `cypressComponents: true` to enable component tests",
-      "cypress": true
+      "cypress": true,
+      "cypressComponents": true
```

```suggestion
      "cypress": true,
      "cypressComponents": true
```

Всюди включений `cypress` - це e2e тести. Якщо поле прибрати, то true всеодно буде значенням за замовчуванням
`cypressComponents` - якщо пишемо component тести ставимо тут true. Якщо тільки їх - виключаємо попередній пункт. `cypressComponents` - за замавчуванням `false`, якщо видалити ключ


Якщо підсумувати - тести, які є включаємо, яких нема - виключаємо.
Мають хоч якісь бути(як зараз заглушки для e2e). Якщо потрібно буде десь виключити зовсім - кажіть - додам підтримку